test/runner_shared.sh

```diff
@@ -81,6 +81,7 @@ ${PSQL} -U ${TEST_PGUSER} \
           sed  -e '/<exclude_from_test>/,/<\/exclude_from_test>/d' \
                -e 's!_[0-9]\{1,\}_[0-9]\{1,\}_chunk!_X_X_chunk!g' \
                -e 's!^ \{1,\}QUERY PLAN \{1,\}$!QUERY PLAN!' \
+               -e 's!:  actual rows!: actual rows!' \
```

Comment:
Hmm not a fan of this. Can you get a stable test output without this change as the number of rows is useful for other tests so I would rather not have this removed globally.

Reply:
I'm keeping the number, just replacing double space for single. The formatting was changed between postgres versions, so I'm normalizing it to avoid having separate references.